backend/migrations/0003_create_user_table.up.sql

```diff
@@ -1,7 +1,6 @@
 -- create_users_table.up
 CREATE TABLE IF NOT EXISTS fluke_user(
     id BIGSERIAL NOT NULL PRIMARY KEY,
-    username TEXT NOT NULL UNIQUE,
```

Comment:
you can't do this.  This will break all existing databases. Once you've committed a migraiton you CANNOT edit it.  To remove a column you must create a new migration that removes this columnm